synapse/lib/aha.py

```diff
@@ -805,6 +805,7 @@ async def setAhaSvcDown(self, name, linkiden, network=None):
     async def _setAhaSvcDown(self, name, linkiden, network=None):
         svcname, svcnetw, svcfull = self._nameAndNetwork(name, network)
         path = ('aha', 'services', svcnetw, svcname)
+        await self.jsonstor.setPathObjProp(path, 'svcinfo/ready', False)
```

Comment:
We may need to check the return value of the cmpDel below to prevent old replay / race / catch up from marking something down?